app/templates/inbox/meetings/new.hbs

```diff
@@ -0,0 +1,70 @@
+{{!-- TODO: Uncomment this once the app uses `{{page-title}}` on all relevant pages --}}
+{{!-- {{page-title (t "inbox.meetings.new.title")}} --}}
+
+<AuModal
```

Comment:
I was planning on extracting a modal component that could be used both here and on the edit page. But while implementing that I noticed that the conditional logic, component arguments, and hacking around required to make it work were not worth it. The template isn't very DRY, but I think simplicity and flexibility this gives is worth it. Now the creation modal can be changed independently. (I don't think all the date fields are needed when creating a meeting for example).